.storybook-sdk/main.js

```diff
@@ -11,7 +11,7 @@ const {
 
 module.exports = {
   stories: ["../enterprise/frontend/src/embedding-sdk/**/*.stories.tsx"],
-  staticDirs: ["../resources/frontend_client"],
+  staticDirs: ["../resources/frontend_client", "./msw-public"],
```

Comment:
Refer to the [installation instruction here](https://storybook.js.org/docs/writing-stories/mocking-data-and-modules/mocking-network-requests). I name the directory like this so it's clear, that's only used for msw.